.github/workflows/sc-client-server-deb10.yml

```diff
@@ -70,6 +70,9 @@ jobs:
       with:
         name: Server Image
         path: sc-server.tar
+    
+    - name: Build Thrift server Docker image
```

Comment:
should be executed conditionally

Reply:
https://github.com/opencomputeproject/SAI-Challenger/blob/424897e208f73953ecac64162235de6044b73ca2/.github/workflows/sc-standalone-deb10.yml#L87

Reply:
Added